src/email/email.js

```diff
@@ -0,0 +1,120 @@
+const { SMTPServer } = require("smtp-server");
+const { MailParser } = require('mailparser');
+const ParseEmailAddress = require("email-addresses");
+const { Logging } = require('../log');
+
+const log = Logging.get("email");
+
+const roomAlias = function(rcptTo, mxDomain) {
```

Comment:
Docstrings would be good here to explain what this does. Also, perhpas this should be `getRoomAliasFromEmailTo` to describe it better?